src/olympia/constants/base.py

```diff
@@ -113,13 +96,13 @@
 ADDON_LPADDON = 6
 ADDON_PLUGIN = 7
 ADDON_API = 8  # not actually a type but used to identify extensions + themes
-ADDON_PERSONA = 9
+_ADDON__PERSONA = 9  # Deprecated - should only exist as STATUS_DELETED
```

Comment:
why the double underscore between `addon` and `persona`?

Reply:
oh, it was just easier to see what usage I still needed to remove/change. I can change it to `_ADDON_PERSONA`.

Reply:
https://github.com/mozilla/addons-server/pull/12031/commits/7ff42233c49c2f18a61900e3466ddb3dacda0730